tests/component-tests/textToSpeech/js/TextToSpeechTests.js

```diff
@@ -31,113 +31,171 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     });
 
     fluid.tests.textToSpeech.cleanUp = function () {
-        speechSynthesis.cancel();
+        if(fluid.textToSpeech.isSupported()) {
```

Comment:
What really fails is my failure to install my precommit hooks to make sure i can't commit unlinted code :(